examples/README.md

```diff
@@ -0,0 +1,30 @@
+# Express examples
+
+This page contains list of examples using Express.
+
+- Authentication with login and passowrd - [examples/auth](./auth)
+- HTTP content negotiation - [examples/content-negotiation](./content-negotiation)
+- Working with cookie-based session - [examples/cookie-sessions](./cookie-sessions)
+- Working with cookies via [`cookie-parser`](https://github.com/expressjs/cookie-parser) - [examples/cookies](./cookies)
+- Files transfer to client - [examples/downloads](./downloads)
+- Working with Embedded JavaScript templating (ejs) - [examples/ejs](./ejs)
+- Example pages for HTTP error - [examples/error-pages](./error-pages)
+- Working with error middleware - [examples/error](./error)
+- Simple request handler - [examples/hello-world](./hello-world)
+- Markdown as template engine - [examples/markdown](./markdown)
+- Working with multiple express routers - [examples/multi-router](./multi-router)
+- Accepting multipart-encoded forms - [examples/multipart](./multipart)
+- MVC style controllers - [examples/mvc](./mvc)
+- Tracking online user activity with `online` and `redis` packages - [examples/online](./online)
+- Working with route parameters - [examples/params](./params)
+- Multiple HTTP operation on same resource - [examples/resource](./resource)
+- Organizing routes using map - [examples/route-map](./route-map)
+- Working with route middleware - [examples/route-middleware](./route-middleware)
+- Organizing routes per each resource - [examples/route-separation](./route-separation)
+- Redis-based search example - [examples/search](./search)
+- Redis-based session - [examples/session](./session)
```

Comment:
```suggestion
- User sessions - [examples/session](./session)
```

The example is less about using Redis and more about sessions in general (which is why there are two different apps in there).